aplib.net-demo/Assets/Scripts/InputManager.cs

```diff
@@ -1,3 +1,4 @@
+using JetBrains.Annotations;
```

Comment:
Perhaps this should be the a unity variant? Don't know, this is probably better.